shared/src/commonMain/kotlin/org/hyperskill/app/learning_activities/domain/model/LearningActivity.kt

```diff
@@ -11,14 +11,23 @@ data class LearningActivity(
     private val stateValue: Int,
     @SerialName("target_id")
     val targetId: Long,
+    @SerialName("target_type")
+    val targetTypeValue: String,
```

Comment:
Instead of using raw string value we can use enum here with case for unsupported types on mobile

Reply:
@vladkash don't see here a problem:

- `LearningActivityTargetType` enum class contains only supported cases
- We are parsing `target_type` as a raw string value (as `state` and `type`)
- If you need to use typed target type value -> `targetType` computed property (also as for `state` and `type`)